Calibration/HcalAlCaRecoProducers/src/AlCaIsoTracksProducer.cc

```diff
@@ -197,11 +196,18 @@ AlCaIsoTracksProducer::AlCaIsoTracksProducer(const edm::ParameterSet& iConfig) :
 
   //create also IsolatedPixelTrackCandidateCollection which contains isolation info and reference to primary track
   produces<reco::HcalIsolatedTrackCandidateCollection>("HcalIsolatedTrackCollection");
```

Comment:
Well - can you approve this wiuthout being made it to a static constant? I have lost the release area already to make this small change now. I shall remember that for the next time. These magic names are there in many places of CMSSW code. Sunanda
________________________________
From: Salvatore Di Guida [notifications@github.com]
Sent: 11 May 2015 14:50
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-alca11 Tune+test several AlCaReco jobs and prepare DQM plots for IsoTrack Trigger (#8943)


In Calibration/HcalAlCaRecoProducers/src/AlCaIsoTracksProducer.cc<https://github.com/cms-sw/cmssw/pull/8943#discussion_r30034843>:

> @@ -197,11 +196,18 @@ AlCaIsoTracksProducer::AlCaIsoTracksProducer(const edm::ParameterSet& iConfig) :
>
>    //create also IsolatedPixelTrackCandidateCollection which contains isolation info and reference to primary track
>    produces<reco::HcalIsolatedTrackCandidateCollection>("HcalIsolatedTrackCollection");


HcalIsolatedTrackCollection appears as a "magic" string in this producer. Should it become a static const data member?

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/8943/files#r30034843>.